functional_tests/features/information_page.feature

```diff
@@ -10,3 +10,14 @@ Feature: A general use information page
         Then the new information page with the added content is displayed
         And the user can see the breadcrumbs
 
+    Scenario: Rich text toolbar is pinned by default
+        Given a CMS user logs into the admin site
+        When the user navigates to the pages menu
+        And the user clicks add child page and chooses information page type
+        Then the rich text toolbar is displayed
+
+    Scenario: Minimap is shown by default
+        Given a CMS user logs into the admin site
+        When the user navigates to the pages menu
+        And the user clicks add child page and chooses information page type
+        Then the minimap is displayed
```

Comment:
These will test the default, but you will need to add a test to ensure the final acceptance criteria passes which is that the user's preference is used if set.
> And if I unpin the toolbar or hide the minimal panel, then that choice is remembered in my browser

Reply:
That was added in [33206a5](https://github.com/ONSdigital/dis-wagtail/pull/106/commits/33206a5773b7559562f0c9976aa5ed17d3c52011) and slightly reworded in [c73178c](https://github.com/ONSdigital/dis-wagtail/pull/106/commits/c73178c9ec364fba8afb6129e474f7c1131c7f89)